src/app/shared/reuseHelper.js

```diff
@@ -52,6 +56,7 @@ module.exports = {
   generateUserDetails: (userDetailsResponse, i18n) => {
     return {
       name: userDetailsResponse.data?.name,
+      formattedName: userDetailsResponse.data?.formattedName,
```

Comment:
Wish we had typescript to make this a bit more obvious what was going on 😢 

Could we rename "name" to "fullName" at least to help distinguish between the two?

Reply:
My reasoning here is to leave as much as possible unchanged since a new field is being added. I did rename it to fullName earlier but change it cause I felt the `name` keeps some consistency with what is sent from core-back too.

Just my thoughts.. But I am happy to be wrong..